docs/src/api-docs.rst

```diff
@@ -1055,3 +1055,81 @@ Body:
       "rating": 'up' | 'down' | null, // up- or downvote (either comment or rating is required)
       "category": String | null,      // comment category ("Technisches Feedback" or null; any other string is treated like null)
    }
+
+Chat
+====
+
+This endpoint provides chat functionality for Integreat app users.
+
+REQUEST
+~~~~~~~
+
+.. code:: http
+
+    GET /api/v3/{region_slug}/{language_slug}/chat/{device_id}/ HTTP/2
+
+.. code:: http
+
+    GET /api/v3/{region_slug}/{language_slug}/chat/{device_id}/{attachment_id}/ HTTP/2
+
+.. code:: http
+
+   POST /api/v3/{region_slug}/{language_slud}/chat/{device_id}/ HTTP/2
+   Content-Type: multipart/formdata
+
+Body:
+
+.. code:: javascript
+
+   {
+      "message": String,               // message the user wishes to send (required)
+      "force_new": Boolean,            // whether to force a new chat instead of continuing existing  (optional)
+   }
+
+
+RESPONSE
+~~~~~~~~
+
+The response to ``POST``-ing to the endpoint is a single object representing
+the message as it is stored in Zammad.
+
+.. code:: javascript
+
+   {
+      "id": Number,                    // message id
+      "body": String,                  // the actual message content
+      "user_is_author": Boolean,       // true if the user sent the message, false otherwise
+      "attachments": [],               // will always be an empty list
+   }
+
+The response to ``GET``-ing the endpoint without an ``attachment_id`` is a list containing all chat messages.
+
+.. code:: javascript
+
+   {
+      "messages" : [                   // A list containing the chat messages
+         "id": Number,                 // message id
+         "body": String,               // the actual message content
```

Comment:
Just a note here: we should be clear that Zammad uses HTML for message formatting.

Reply:
Good point, will add a note.